The old project was synchronized with the Spring Framework. The produ…

awestor opened the pull request:
…ct entity class, classes and interfaces, project repositories and services, as well as console input and command processing classes were added.
В старый проект были добавлены файлы из сгенерированного проекта готового к работе с Spring Framework. Были Добавлены классы для работы с сущностью Product, а также классы для работы с консольным вводом.

awestor committed:
The old project was synchronized with the Spring Framework. The product entity class, classes and interfaces, project repositories and services, as well as console input and command processing classes were added.

LARCH1K reviewed (commented):
Всё сделано хорошо, присутствуют немного мелких замечаний

P.S. Круто что ты подробно называешь коммит. Только формируй его пожалуйста так: в первой строке короткое, но понятное название коммита, дальше пустая строка, дальше уже описание любой длины

P.P.S. мне в целом не принципиально на каком языке коммит, но на всякий уточню, что можно и на русском (пиши как хочешь, мне нормально и так)

awestor committed:
the comments have been corrected.

a class has been added containing methods for entering values from the console. he avaoc has also been updated and the logic for updating product field values has been changed.
